.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Overview
+<!-- Required. Describe, briefly, the behavior experienced and desired. -->
+
+## Screenshots/Screencasts
+<!-- Optional. Attach screenshot(s) and/or screencast(s) that demo the behavior. -->
+
+## Steps to Recreate
+<!-- Required. List exact steps (numbered list) to reproduce errant behavior. -->
+
+## Environment
+<!-- Required. What is your operating system, software version(s), etc. -->
```

Comment:
Nit: I guess the operating system won't be an issue 99.99% of the time:slightly_smiling_face: Thoughts on leaving that out of the required information?

Reply:
Something like that sounds good to me 👍🏼

Reply:
:+1:

Reply:
:bow: Amended.